### PR TITLE
Add "SASL wrapping" field to AD directory params.

### PR DESCRIPTION
**Pull request by jceel**
Ticket: #19252

### CODE REVIEW

File: ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html [hunk 1]
```diff
@@ -88,6 +88,23 @@
             }
         },
 
+        "saslWrapping": {
+            "prototype": "blue-shark/ui/field-select.reel",
+            "properties": {
+                "element": {"#": "saslWrapping"},
+                "label": "SASL Wrapping",
+                "options": [
```
Comment (thibaultzanini) [inline ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html:96]: looks like you could require the enum `LdapDirectoryParamsSaslWrapping` within the controller and bind the `options` property of the select component to the `members` property of the enum.

**Review by thibaultzanini — CHANGES_REQUESTED** [review]
Just a little change required otherwise looks good to me :)

File: ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html (hunk 1 above)
Comment (jceel) [reply ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html:96]: Please show me some example.

Comment (pchaussalet) [reply ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html:96]: you can refer to this as an example: https://github.com/freenas/gui/blob/master/ui/sections/system/inspectors/alert.reel/table-predicates.reel/table-row-predicate.reel/table-row-predicate.js#L29

**Pull request closed without merging**